src/config/deploy.js

```diff
@@ -139,3 +139,16 @@ module.exports.addProfileProperties = config => {
 
   return combine(config, profileProps);
 };
+
+module.exports.getCredentials = () => {
```

Comment:
The config resolution is quite complex so it might be best if @colingourlay could advise, but it would be nice if credentials were fetched the same way as for deploy. For example, it seems from https://github.com/abcnews/aunty/blob/main/src/config/deploy.js#L123 that it's possible to set the profiles file path to something other than the default for the purposes of deploy and we wouldn't want to end up with a mismatch here.

Reply:
Yes this would be nice. @colingourlay I might grab you to run through a few things when I can.

Reply:
Same with this, should be able to get it from `getDeployConfig` from what @colingourlay tells me.